tests/transform_dialect/cuda/vecadd2d_codegen_spec.mlir

```diff
@@ -0,0 +1,12 @@
+transform.structured.canonicalized_sequence failures(propagate) {
+^bb1(%varop: !pdl.operation):
+  %generic = transform.structured.match ops{["linalg.generic"]} in %varop
+  %foreach, %tiled = transform.iree.tile_to_foreach_thread_and_workgroup_count_region %generic tile_sizes [4, 4] ( mapping = [#gpu.block<z>, #gpu.block<x>] )
```

Comment:
I am going to insist that you use different tile sizes here.
If `2, 3` is too annoying to FileCheck, please choose another set of different values.